mdoc/Mono.Documentation/Util/AttachedEntitiesHelper.cs

```diff
@@ -180,6 +180,28 @@ private static bool IsAssignableTo(TypeReference type, string targetTypeName)
             return type.FullName == targetTypeName || IsAssignableTo(typeDefenition.BaseType, targetTypeName);
         }
 
+        private static bool ChkFieldVisible(PropertyDefinition field)
```

Comment:
This should still be called `ChkPropertyVisible` ... and you should add **another**, separate, method called `ChkFieldVisible` to service line 130 above. This new method should accept a `FieldDefinition`, and it won't have the same implementation (it will be much simpler) as the property one ... because a field is just a field, so you don't have to check any other methods. You can basically just verify to see if the field itself is public or is explicitly implemented